VirtualReality/MRML/vtkMRMLVirtualRealityViewNode.cxx

```diff
@@ -41,7 +42,7 @@ vtkMRMLVirtualRealityViewNode::vtkMRMLVirtualRealityViewNode()
   , MotionSensitivity(0.5)
   , ControllerTransformsUpdate(false)
   , ControllerModelsVisible(true)
-  , TrackerReferenceModelsVisible(true)
+  , LighthouseReferenceModelsVisible(true)
```

Comment:
I'm not sure if LighthouseReferenceModels makes much sense. If we change the name then it should be LighthouseModels.